README.md

```diff
@@ -1,2 +1,64 @@
-# scraping-placard
+# scraping-placard :soccer: :basketball: :rugby_football: :tennis:
 NodeJS service that fetches the Placard website, extracts events and bets from the HTML responses and stores the data in a database.
+
+~~~EU APENAS ESCREVO EM PORTUGUES!!! KOSSAK31~~
+
+## TOOLS
+* website <https://www.jogossantacasa.pt/web/Placard>
+* nodejs
+* mongodb
+---
+
+
+# Project tree
+* [fetch](./fetch)
+  * [scrapers](./fetch/scrapers)
+    * [getEventsGroupedByCompetition.js](./fetch/scrapers/getEventsGroupedByCompetition.js)
+    * [getSportsAndCountries.js](./getSportsAndCountries.js)
+  * [transformers](./fetch/transformers)
+    * [arrayBufferToString.js](./fetch/transformers/arrayBufferToString.js)
+   	* [iso88591ToUTF8.js](./fetch/transformers/iso88591ToUTF8.js)
+  * [index.js](./fetch/index.js)
+* [models](./models)
+    * [bet.js](./models/bet.js)
+    * [competition.js](./models/competition)
+    * [country.js](./models/country.js)
+    * [event.js](./models/event.js)
+    * [nameValue.js](./models/nameValue.js)
+    * [sport.js](./models/sport.js)
+* [storage](./storage)
+    * [index.js](./index.js)
+* [.gitignore](./.gitignore)
+* [README.md](./README.md)
+* [index.js](./index.js)
+* [package.json](./package.json)
+* [start.js](./start.js)
```

Comment:
This is cool but we have to change this every time we add/remove/move a file. I'd prefer to avoid this files/folders structure. Maybe we can have some important files listings next to an explanation of its importance but I consider it as optional!